src/graph/FindPathExecutor.cpp

```diff
@@ -598,6 +599,13 @@ FindPathExecutor::getStepOutProps(bool reversely) {
     }
     std::vector<storage::cpp2::PropDef> props;
     for (auto &e : *edges) {
+        {
+            storage::cpp2::PropDef pd;
+            pd.owner = storage::cpp2::PropOwner::EDGE;
+            pd.name = "_dst";
+            pd.id.set_edge_type(e);
+            props.emplace_back(std::move(pd));
+        }
```

Comment:
If `_dst` would return by default, then this property is no more required in prop list.